phpunit/functional/ToolboxTest.php

```diff
@@ -1188,7 +1188,7 @@ public static function urlProvider()
             ['https://localhost?test=true&othertest=false', true],
             ['https://localhost/front/computer.php?is_deleted=0&as_map=0&criteria[0][link]=AND&criteria[0][field]=80&criteria[0][searchtype]=equals&criteria[0][value]=254&search=Search&itemtype=Computer', true],
             ['https://localhost?test=true&#38;othertest=false', true], /* sanitized URL, &#38; is & */
-            ['https://localhost?test=true&#39;othertest=false', false], /* any entity */
```

Comment:
wgy removing this one?

Reply:
Symfony's regex says it is valid and it looks correct to me too. No reason an apostrphe couldn't be here, it just means the "othertest=false" isn't a separate parameter.